features/202-session-management/api.md

```diff
@@ -2,15 +2,28 @@
 
 ## SDK API
 ```typescript
+interface UserAgent {
+    raw: string;
+    name: string;
+    version: string;
+    os: string;
+    osVersion: string;
+    app: string;
+    appVersion: string
+    device: string;
+    deviceID: string;
+}
+
 interface Session {
     id: string;
     identityID: string;
     createdAt: Date;
     lastAccessedAt: Date;
     createdByIP: string;
     lastAccessedIP: string;
-    createdByUserAgent: string;
-    lastAccessedUserAgent: string;
+    createdByUserAgent: UserAgent;
+    lastAccessedUserAgent: UserAgent;
```

Comment:
is it "normal" for a session to be used by different UserAgent? I know it is technically possible, but if that happens should that be treated as an "exception" (so maybe just log it down?) or it is "normal" thing that we will have `lastAccessedUserAgent`?

Reply:
I don't think a changed raw user agent string is exceptional case, e.g. updating OS / browser. Including a `lastAccessedUserAgent` would not have observable overhead, so I think it is acceptable.

Reply:
Acceptable but still feel like something weird / not useful..?

Reply:
It would be more consistent: all transient attributes (e.g. timestamp, IP, user agent) would have value at creation and at last access.

Reply:
fair enough.